core/src/main/java/org/mule/processor/chain/InterceptingChainLifecycleWrapperPathSkip.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.processor.chain;
+
+import org.mule.api.processor.MessageProcessor;
+import org.mule.api.processor.MessageProcessorChain;
+
+import java.util.List;
+
+public class InterceptingChainLifecycleWrapperPathSkip extends InterceptingChainLifecycleWrapper
```

Comment:
Why do some uses of InterceptingChainLifecycleWrapper need to be skipped but others not?